qiskit/transpiler/__init__.py

```diff
@@ -36,84 +36,165 @@
+    non-polynomial in complexity, and we need to ensure we finish the job in a workable amount of
+    time.
+
+Choosing Preset Stage Implementations
+-------------------------------------
+
+Qiskit includes several implementations of several of the above stages, and more can be installed as
+separate "plugins".  To control which implementation of a stage is used, pass its name to the
+``<stage>_method`` keyword argument of the two functions, such as
+``translation_method="translator"``.  To read more about implementing such external plugins for a
+stage, see :mod:`qiskit.transpiler.preset_passmanagers.plugin`.
+
+For example, to generate a preset pass manager at optimization level 1 that explicitly uses the
+``trivial`` method for layout with the ``sabre`` method for routing, we would do:
 
 .. plot::
-   :include-source:
-   :nofigs:
+    :include-source:
+    :nofigs:
 
-    from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
+    from qiskit.transpiler import generate_preset_pass_manager
     from qiskit.providers.fake_provider import GenericBackendV2
 
+    # Whatever backend you like:
     backend = GenericBackendV2(num_qubits=5)
-    pass_manager = generate_preset_pass_manager(3, backend)
-
-which will generate a :class:`~.StagedPassManager` object for optimization level 3
-targeting the :class:`~.GenericBackendV2` backend (equivalent to what is used internally
-by :func:`~.transpile` with ``backend=GenericBackendV2(5)`` and ``optimization_level=3``).
-You can use this just like you would any other :class:`~.PassManager`. However,
-because it is a :class:`~.StagedPassManager` it also makes it easy to compose and/or
-replace stages of the pipeline. For example, if you wanted to run a custom scheduling
-stage using dynamical decoupling (via the :class:`~.PadDynamicalDecoupling` pass) and
-also add initial logical optimization prior to routing, you would do something like
-(building off the previous example):
+
+    pass_manager = generate_preset_pass_manager(
+        optimization_level=1,
+        backend=backend,
+        layout_method="trivial",
+        routing_method="sabre",
+    )
+
+.. note::
+
+    The built-in set of available plugins for each stage is part of Qiskit's public API, and subject
+    to all the stability guarantees.  This includes the high-level logical effects of that method
+    (for example, ``routing_method="sabre"`` will always use a Sabre-derived algorithm).  The exact
+    internal construction of the :class:`.PassManager` representing the stage is not, however; the
+    order of passes might change between minor versions, or new passes might be introduced.
+
+    For any stage that has one, the method named ``"default"`` is most subject to change.  We will
+    typically only make complete algorithmic changes in the default method across a major-version
+    boundary, but we may well rebalance heuristics and add new passes to default methods between
+    minor versions.
```

Comment:
Here, "we" clearly refers to the Qiskit team - is it appropriate to substitute "Qiskit" here? "Qiskit will typically only make....., but it may well rebalance....."

Reply:
Changed to "Qiskit" as the subject rather than "us".